web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -239,11 +239,6 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
 
         params.onAccept = function(callbackParams) {
             self.downloader.download(collection, generateUrlCallbackScope, generateUrlCallback, callbackParams);
-            trackDownloadUsage(
-                OpenLayers.i18n('downloadTrackingActionPrefix') + OpenLayers.i18n(textKey),
-                collection.getTitle(),
-                undefined
-            );
         };
```

Comment:
I reckon this is the right place to track download usage.  Otherwise we are burying this in a downloader, or a callback to generate the download url.  This doesn't seem like the right place to put UI action tracking.

I'd leave it here.

To include collection filter information applicable for the download we could just

a) add the url generated by the callback as the tracking details, 
b) include the filters applied to the collection at this point,
c) if you want the filters formatted just loop through the filters getting the human readable form

I don't think we need to format tracking related info in the download handler as it isn't tied to that and I don't think we should use the insertion service
which seems to be designed to generate formatted filter details for insertion into the download page.  If we use that then whenever we change the format there we would have 
to change getCollectionFiltersAstext to remove the page formatting.  I'd have a look at Portal.filter.combiner.HumanReadableFilterDescriber which is what the insretion services uses under the hood to
get humanreadable filter descriptions